code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -572,32 +564,46 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 		else
 			shielding_powered = FALSE
 
+///return the icon_state for the canister's indicator light based on its current pressure reading
```

Comment:
it's not an icon state it's an icon state component

Reply:
Yup ill update that

Reply:
Done